src/zeit/content/article/breaking.py

```diff
@@ -37,3 +41,28 @@ def banner_matches(self, banner):
 @grok.implementer(zeit.connector.interfaces.IWebDAVProperties)
 def webdav_properties(context):
     return zeit.connector.interfaces.IWebDAVProperties(context.context)
+
+
+class BreakingNewsArticle(zeit.content.article.article.Article):
+    """BreakingNewsArticle is a special article which triggers the breaking
+    news banner.
+    """
```

Comment:
Docstrings sollten in der ersten Zeile kurz beschreiben worum es geht, sich aber nicht auf mehrere Zeilen ausweiten. Sind mehr Infos notwendig, dann Leerzeile und beliebig viel weiteren Text.